src/test/java/com/browserstack/test/suites/offers/OfferTest.java

```diff
@@ -0,0 +1,27 @@
+package com.browserstack.test.suites.offers;
+
+import com.browserstack.test.suites.TestBase;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+public class OfferTest extends TestBase {
+
+    @Override
+    public String getGeolocation() {
+        return "IN";
+    }
```

Comment:
Can we check with Gokul and UI team if the application will be using GPS or IP location going forward.
We need to have the OffersTest enabled in either case. If app is expected to change, then for now let the test fail.

If the app logic is frozen and we are going with GPS location, then implementing the mock location as in other repos is the only option.

Based on what I see in the latest app code: It seems to be using GPS location with the cities as mentioned in the below code.
https://github.com/browserstack/browserstack-demo-app/blob/master/pages/api/offers.js

Reply:
@samiran-bs  Application will be using browser's geolocation feature